components/function-controller/internal/webhook/validating_webhook_test.go

```diff
@@ -98,6 +99,7 @@ func TestValidatingWebHook_Handle(t *testing.T) {
 			},
 			args: args{
 				ctx: context.Background(),
+				//nolint
 				req: admission.Request{
```

Comment:
We can add the field name, because it will take longer investigating why `nolint` was added.